template/build/webpack.dev.conf.js

```diff
@@ -28,7 +28,8 @@ module.exports = merge(baseWebpackConfig, {
     new HtmlWebpackPlugin({
       filename: 'index.html',
       template: 'index.html',
-      inject: true
+      inject: true,
+      serviceWorkerLoader: ''
```

Comment:
An alternative to loading nothing would be to load a no-op service worker file, like https://github.com/facebookincubator/create-react-app/pull/2276/commits/8b426ad9a9c71d5113baa367bb0d25ff0bce0567

That works around an issue where someone might accidentally start up their prod environment on the port that they normally use for dev on localhost, causing the prod service worker to stick around indefinitely (see https://github.com/facebookincubator/create-react-app/issues/2272).